test-data/unit/check-super.test

```diff
@@ -366,13 +370,22 @@ class C(B):
 [case testSuperInMethodWithNoArguments]
 class A:
     def f(self) -> None: pass
+    @staticmethod
+    def st() -> int:
+        return 1
 
 class B(A):
     def g() -> None: # E: Method must have at least one argument. Did you forget the "self" argument?
-        super().f() # E: super() requires one or more positional arguments in enclosing function
+        super().f() # E: "super" requires one or more positional arguments in enclosing function
     def h(self) -> None:
         def a() -> None:
-            super().f() # E: super() requires one or more positional arguments in enclosing function
+            super().f() # E: "super" requires one or more positional arguments in enclosing function
+    @staticmethod
+    def st() -> int:
+        reveal_type(super(B, B).st())  # N: Revealed type is "builtins.int"
+        super().st() # E: "super" requires one or more positional arguments in enclosing function
```

Comment:
Here again, it's specifically the zero-argument form of `super()` that we're complaining about (the `reveal_type()` call in the line immediately above this one works fine for the two-argument form of `super()`). So I'd keep the `()` here, personally

Reply:
And it should be "one or two" not "one or more" 👍